book/release-service.md

```diff
@@ -39,8 +28,95 @@ The diagram below shows the flow of custom resources and the orchestration of pi
+
+### UPDATE
+
+| Custom Resource            | When?                                                    | Why?                                                         |
+|----------------------------|----------------------------------------------------------|--------------------------------------------------------------|
+| SnapshotEnvironmentBinding | When deploying the Snapshot to an Environment            | Update exiting SnapshotEnvironmentBinding for a new Snapshot |
+| Release                    | During the lifecycle of an attempt to release a Snapshot | Provide status for the execution of the Release Pipeline     |
+
+### WATCH
+
+| Custom Resource              | When?                                                   | Why?                                                            |
+|------------------------------|---------------------------------------------------------|-----------------------------------------------------------------|
+| PipelineRun                  | Once Pipeline created                                   | Relay the Release PipelineRun status to the Release for viewing |
+| SnapshotEnvironmentBinding   | After the SnapshotEnvironmentBinding created or updated | Relay the GitOps deployment status to the Release for viewing   |
+
+#### Annotations/Labels
+
+Following the [annotation guidelines](https://docs.google.com/document/d/1gyXM3pkKFFfxHZnopBi_53vREFWhwA0pFUuIhopDuEo/edit#), the Release Service sets the below annotations on PipelineRuns.
+```
+“pipelines.appstudio.openshift.io/type":     "release",
+"appstudio.openshift.io/snapshot":           “<snapshot name>”,
+“appstudio.openshift.io/component":          “<component name>”,
+“appstudio.openshift.io/application":        “<application name>”,
+"release.appstudio.openshift.io/name":       "<release name>",
+"release.appstudio.openshift.io/namespace":  "<managed workspace name>"
+```
+The Release service will copy the annotations and labels from the Release CR and append those to the Release PipelineRuns for traceability across the system per [Labels and Annotations for StoneSoup pipelines](https://docs.google.com/document/d/1fJq4LDakLfcAPvOOoxxZNWJ_cuQ1ew9jfBjWa-fEGLE/edit#) and [StoneSoup builds and tests PRs](https://docs.google.com/document/d/113XTplEWRM63aIzk7WwgLruUBu2O7xVy-Zd_U6yjYr0/edit#)
+
+## Detailed Workflow
+
+1. A Release CR is created automatically by the Integration Service or manually by a user.
+   * The `spec.snaphot` should reference the Snapshot to release.
+   * The `spec.releasePlan` should reference the ReleasePlan to use.
+2. Find the matching ReleasePlanAdmission in the `spec.target` namespace specified in the ReleasePlan.
+3. Extract the `spec.releaseStrategy` from the ReleasePlanAdmission
+4. Extract the `spec.serviceAccount`, `spec.bundle`, `spec.pipeline` and `spec.policy` to use from the Release Strategy.
```

Comment:
Based on consistency, I think it should be `ReleaseStrategy`, not `Release Strategy`